projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/common/DataJobDeploymentExtension.java

```diff
@@ -142,8 +142,16 @@ public void beforeEach(ExtensionContext context) throws Exception {
                       .with(user(USER_NAME))
                       .content(jobZipBinary)
                       .contentType(MediaType.APPLICATION_OCTET_STREAM))
-              .andExpect(status().isOk())
               .andReturn();
+
+      if (uploadResult.getResponse().getStatus() != 200) {
```

Comment:
Is it possible to get a 201 here, for example?

Reply:
Nope only 200 is possible.